editor/editor_inspector.cpp

```diff
@@ -2878,28 +2848,12 @@ void EditorInspector::update_tree() {
 			}
 
 			// Set the category label.
-			category->label = label;
+			category->label = doc_name;
```

Comment:
Not sure why this was changed, this changes the behavior as `label` can be updated to match the script name.

Reply:
`doc_name` is already the script name, but I see what you mean.

Reply:
I suspect there may be some duplication of logic in the code above, but it's a such a small and unnecessary change for this PR, I'd rather avoid it to not introduce possible regressions. Getting these categories to behave took a lot of time and PRs after all.